layrry-config/src/main/java/org/moditect/layrry/config/LayersConfigParser.java

```diff
@@ -0,0 +1,28 @@
+/**
+ *  Copyright 2020 The ModiTect authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.moditect.layrry.config;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+
+public interface LayersConfigParser {
```

Comment:
Could you add a class comment, given it's public API/SPI?

Reply:
Sure thing.